woocommerce/woocommerce
# Collapsible panel for shipping calculator in cart block

### Changes proposed in this Pull Request:

Closes #50554

1. This moves the shipping calculator into a collapsible panel
2. This adjusts copy in several places
3. Where before some of the copy for different states (no address etc) was in different places, now it is mostly consolidated to the collapsible calculator panel button title.
4. Always show the rates even when the address panel is expanded.

See [the designs](https://www.figma.com/design/c8VYjXqluE9vlVSE1RgJix/%5BPublic%5D-Cart-%26-Checkout-Blocks?node-id=463-3041&node-type=canvas&t=dr5VOEQYHq4M0amY-0) for more details.


Examples of different states:

<img width="453" alt="Screenshot 2024-11-12 at 2 41 31 PM" src="https://github.com/user-attachments/assets/c94bedd3-420e-4b1b-9697-e247c811f97e">

<img width="457" alt="Screenshot 2024-11-12 at 2 44 43 PM" src="https://github.com/user-attachments/assets/35205a1d-5248-43af-bfd6-5e199dd00f87">

<img width="418" alt="Screenshot 2024-11-12 at 2 42 55 PM" src="https://github.com/user-attachments/assets/19d38156-348a-4567-bf27-feba03cb52d1">




### How to test the changes in this Pull Request:

1. Test with various settings enabled/disabled (ie disable/enable the cart calculator and look for regressions)
2. Test entering addresses to check delivery options, switching rates etc
3. Visual test, have a look at the designs and see that the look and feel here matches the design and that copy looks correct.

### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Cart block: display shipping calculator in collapsible panel design.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
